CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 7.1.1 (March 2, 2023). Tested on Artifactory 7.55.2
+
+BUG FIXES:
+* resource/artifactory_remote_docker_repository, resource/artifactory_remote_helm_repository: fixed the issue when `externalDependenciesEnabled` was impossible to update. 
```

Comment:
Let's refer to the TF attribute (vs API field).

Reply:
Still has `externalDependenciesEnabled` here in line above 😄

Reply:
well :) okay, fixed!